bmorph/core/workflows.py

```diff
@@ -6,138 +6,36 @@
-                                   pd.to_datetime('{}-09-30 00:00:00'.format(year)))
-        raw_bmorph_window =  slice(pd.to_datetime('{}-01-01 00:00:00'.format(year)),
-                                   pd.to_datetime('{}-12-31 00:00:00'.format(year)))
-        raw_cdf_window = slice(pd.to_datetime('{}-01-01 00:00:00'.format(year - overlap_period)),
-                               pd.to_datetime('{}-12-31 00:00:00'.format(year + overlap_period)))
-        if (raw_cdf_window.start < raw_ts_window.start):
-            offset = raw_ts_window.start - raw_cdf_window.start
-            raw_cdf_window = slice(raw_ts_window.start, raw_cdf_window.stop + offset)
-        if(raw_cdf_window.stop > raw_ts_window.stop):
-            offset = raw_ts_window.stop - raw_cdf_window.stop
-            raw_cdf_window = slice(raw_cdf_window.start + offset, raw_ts_window.stop)
-
-        bc_total, bc_mult = bmorph.bmorph(raw_ts, raw_cdf_window, raw_bmorph_window, obs_ts, train_ts,
-                                          train_window, n_smooth_short, raw_y, obs_y, train_y,
-                                          bw=bw, xbins=xbins, ybins=ybins, rtol=rtol, atol=atol,
-                                          method=method)
-        bmorph_ts = bmorph_ts.append(bc_total)
-        bmorph_multipliers = bmorph_multipliers.append(bc_mult)
-
-
-    # Apply the correction
-    if n_smooth_long:
-        nrni_mean = obs_ts[reference_window].mean()
-        train_mean = train_ts[reference_window].mean()
-        bmorph_corr_ts, corr_ts = bmorph.bmorph_correct(raw_ts, bmorph_ts, raw_ts_window,
-                                                        nrni_mean, train_mean,
-                                                        n_smooth_long)
-    else:
-        bmorph_corr_ts = bmorph_ts
-    return bmorph_corr_ts[bmorph_window], bmorph_multipliers[bmorph_window]
-
-def apply_interval_bmorph(raw_ts, train_ts, obs_ts,
-        train_window, bmorph_window, reference_window, bmorph_interval,
-        bmorph_overlap, n_smooth_long=None, n_smooth_short=5,
-        raw_y=None, train_y=None, obs_y=None, bw=3, xbins=200, ybins=10,
-        rtol=1e-6, atol=1e-8, method='hist', **kwargs):
-    """Bias correction is performed by bmorph on user-defined intervals.
-
-    Parameters
-    ----------
-    raw_ts : pandas.Series
-        Raw flow timeseries.
-    train_ts : pandas.Series
-        Flow timeseries to train the bias correction model with.
-    obs_ts : pandas.Series
-        Observed/reference flow timeseries.
-    train_window : pandas.date_range
-        Date range to train the bias correction model.
-    bmorph_window : pandas.date_range
-        Date range to apply bmorph onto flow timeseries.
-    reference_window : pandas.date_range
-        Date range to smooth elements in 'raw_ts' and 'bmorph_ts'.
     bmorph_interval : pandas.DateOffset
         Difference between bmorph application intervals.
     bmorph_overlap : int
```

Comment:
to avoid the previous error, should be specified here that the overlap is in days

Reply:
same for `blendmorph`

Reply:
Done